packages/pricing/src/migrations/Migration20240312163523.ts

```diff
@@ -0,0 +1,18 @@
+import { Migration } from "@mikro-orm/migrations"
```

Comment:
question: Can we move this to the master migration?

Reply:
yeah, this was supposed to be removed. my bad!